docs/source/caching/normalized-cache.mdx

```diff
@@ -172,31 +176,29 @@ val response = apolloClient.query(query)
 
   // Always use the network and never check the cache, even if network fails
   .fetchPolicy(FetchPolicy.NetworkOnly)
-
-  // Execute the query
-  .execute()
-```
-The `CacheAndNetwork` policy can emit multiple values, so you call `toFlow()` instead of `execute()`:
-
-```kotlin
-apolloClient.query(query)
-
-  // Check the cache and also use the network (1 or 2 values can be emitted)
+  
+  // Check the cache and also use the network
   .fetchPolicy(FetchPolicy.CacheAndNetwork)
 
   // Execute the query and collect the responses
   .toFlow().collect { response ->
-      // ...
+    // ...
   }
 ```
+Note that cache misses **will** emit a response with a non null `.exception`, meaning that some of these policies can emit multiple values:
+- `CacheFirst` can emit 1 or 2 values
+- `NetworkFirst` can emit 1 or 2 values
+- `CacheAndNetwork` will emit 2 values
+
+For that reason you should use `.toFlow()` rather than `.execute()` when using these policies.
 
 As with `normalizedCache(NormalizedCacheFactory)`, `fetchPolicy(FetchPolicy)` is an extension function on `ApolloClient.Builder()`, so you need `apollo-normalized-cache` in your classpath for this to work.
 
 Because the normalized cache deduplicates data, it enables you to react to cache changes. You do this with `watchers` that listen for cache changes. [Learn more about query watchers.](./query-watchers/)
 
-## Specifying cache IDs
+## Specifying cache keys
```

Comment:
Sooooo the docs are quite heterogeneous about this.... 

Web has mostly "cache IDs", iOS has both "cache IDs" and "cache keys (not sure what the difference is). Given that most of the normalized cache documentation/content out there is for web (therefore using "cache IDs"), I would exercise caution before changing this.